vector/src/main/AndroidManifest.xml

```diff
@@ -140,6 +140,7 @@
                 <data
                     android:host="connect"
                     android:scheme="element" />
+                <data android:scheme="schildichat" />
```

Comment:
We'd rather not touch that one, as I don't care enough about SSO to find proper ways to test it. Also, you didn't touch the files handling SSO connect things, so this might not work either way.